src/index.html

```diff
@@ -7,8 +7,90 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img
+        src="images/logo.png"
+        class="logo"
+        alt="home">
+      </a>
+
+      <nav class="navigation">
+        <ul class="fullList">
+          <li class="list">
+            <a href="apple"
```

Comment:
Change the href value to a valid URL or use # as a placeholder.

Same for the rest of nav links.